dev/bots/test.dart

```diff
@@ -76,8 +76,8 @@ Future<Null> _verifyInternationalizations() async {
   final String localizationsFile = path.join('packages', 'flutter_localizations', 'lib', 'src', 'l10n', 'localizations.dart');
 
   final EvalResult sourceContents = await _evalCommand(
-    'cat',
-    <String>[localizationsFile],
+    'powershell',
```

Comment:
This will not work on mac/linux now?

Reply:
Oops, yes, this has to be guarded as OS(windows)-specific